### DISCUSSION TITLE
Unauthorized cache warning

### DISCUSSION THREAD

**Discussion by dcantu96** [vercel/turborepo#6740] [Help] [answered]
Hello I just updated from 1.10.16 to 1.11.0 and I started to see the warning below when I ran a turbo command. Some other helpful notes are that our company hosts the remote cache servers and that the TURBO_TOKEN is set in the root `.env` file. It's been working without warnings until now

```
 WARNING  artifact verification failed: Error making HTTP request: HTTP status client error (401 Unauthorized) for url (https://turborepo.company.app/v8/artifacts/123ID?teamId=MY_TEAM_ID)
```

**Comment by spacedawwwg** [comment]
Just started searching for the exact same issue and came across this discussion. Wonder if it might be better as an issue @dcantu96?

**Reply by dcantu96** [reply]
It might be, I set it as a discussion because maybe our setup is missing something? I didn't find anything on the docs about this

**Reply by anthonyshew** [reply]
https://github.com/vercel/turbo/discussions/6740#discussioncomment-7823482

**Comment by spacedawwwg** [comment]
Reverting back to 1.10 for now

**Comment by anthonyshew** [comment, marked as answer]
This is a bug. Please move discussion over to here: https://github.com/vercel/turbo/issues/6765
